core/components/com_members/admin/controllers/members.php

```diff
@@ -159,8 +165,18 @@ public function displayTask()
 				$note
 					->select('id')
 					->select('user_id');
+			}])
+			->including(['auths', function ($auth) {
+				$auth
+					->select('*');
 			}]);
 
+		if ($filters['status'])
```

Comment:
Since I just realized that if the key to relate jos_users_log_auth and jos_users is 'username' instead of 'user_id'